app/src/main/java/com/google/droidjump/models/Droid.java

```diff
@@ -81,15 +81,15 @@ public int getJumpHeight() {
     }
 
     public void useJumpingBitmap() {
-        bitmap = getDroidTypes()[GameConstants.DROID_JUMPING_CHARACTER_INDEX];
+        picture = getDroidTypes()[GameConstants.DROID_JUMPING_CHARACTER_INDEX];
```

Comment:
nit: I don't think you need this method here ("getDroidTypes"). Just use droidTypes directly.

Reply:
here and below.